keyboards/yetis/keyboard.json

```diff
@@ -0,0 +1,102 @@
+{
+    "manufacturer": "jogme",
+    "keyboard_name": "Yetis",
+    "maintainer": "qmk",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "development_board": "elite_c",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "direct": [
+            [null, null, "F7", "F6", "F5", "F4"],
+            ["F0", "F1", "B1", "B3", "B2", "B6"],
+            [null, "C7", "D5", "B7", "B5", "E6"],
+            [null, null, null, null, "D7", "C6"]
+        ]
+    },
+    "rgblight": {
+        "animations": {
+            "breathing": true,
+            "knight": true
+        },
+        "led_count": 12,
+        "split": true,
+        "split_count": [6, 6]
+    },
+    "split": {
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "direct": [
```

Comment:
It looks like this is identical but flipped to the left/main config? 
If that's the case, you don't actually need this block, and you can switch the column order for the right half in the layout section below.

Reply:
I don't understand how can I do it. Can you please explain in more detail?